sdkproject/Assets/Mapbox/Unity/Editor/PopupSelectionMenu.cs

```diff
@@ -157,6 +160,11 @@ public void AddNewInstanceToArray(object obj)
 					_finalize.serializedObject.ApplyModifiedProperties();
 				}
 			}
+			MapboxDataProperty mapboxDataProperty = (MapboxDataProperty)EditorHelper.GetTargetObjectOfProperty(_container);
```

Comment:
I believe so - will test and update if so

Reply:
The requested change does not work. This is because CheckForModified property will only set HasChanged to true if property.serializedObject.ApplyModifiedProperties() returns true; in this case, it does not, so HasChanged is not set to true, thus no update event fires.